invenio_app_ils/ui/backoffice/src/pages/LoanDetails/state/tests/actions.test.js

```diff
@@ -0,0 +1,159 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+// Actions to be tested
+import * as actions from '../actions';
+import { initialState } from '../reducer';
+import { serializeLoanDetails } from '../selectors';
+import * as types from '../types';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+const loansBaseUrl = 'https://localhost:5000/api/circulation/loans';
```

Comment:
`baseUrl = https://localhost:5000/api/` and put this somewhere in common to all tests?